frontend/public/components/namespace.jsx

```diff
@@ -884,26 +904,28 @@ export const NamespaceSummary = ({ ns }) => {
       </div>
       <div className="col-sm-6 col-xs-12">
         <dl className="co-m-pane__details">
-          <DetailsItem label="Status" obj={ns} path="status.phase">
+          <DetailsItem label={t('namespace~Status')} obj={ns} path="status.phase">
             <Status status={ns.status.phase} />
           </DetailsItem>
           {canListSecrets && (
             <>
-              <dt>Default Pull Secret</dt>
+              <dt>{t('namespace~Default Pull Secret')}</dt>
               <dd>
                 <PullSecret namespace={ns} />
               </dd>
             </>
           )}
-          <dt>Network Policies</dt>
+          <dt>{t('namespace~Network Policies')}</dt>
           <dd>
-            <Link to={`/k8s/ns/${ns.metadata.name}/networkpolicies`}>Network Policies</Link>
+            <Link to={`/k8s/ns/${ns.metadata.name}/networkpolicies`}>
+              {t('namespace~Network Policies')}
+            </Link>
           </dd>
           {serviceMeshEnabled && (
             <>
-              <dt>Service Mesh</dt>
+              <dt>{t('namespace~Service Mesh')}</dt>
               <dd>
-                <GreenCheckCircleIcon /> Service Mesh Enabled
+                <GreenCheckCircleIcon /> {t('namespace~Service Mesh Enabled')}
```

Comment:
```suggestion
                <GreenCheckCircleIcon /> {t('namespace~Service mesh enabled')}
```

Not new, but the checkbox here is missing aria attributes.